var/spack/repos/builtin/packages/mvapich-plus/package.py

```diff
@@ -332,19 +335,17 @@ def configure_args(self):
             args.extend(
                 [
                     "--enable-cuda",
-                    "--with-cuda={0}".format(spec["cuda"].prefix),
-                    "NVCCFLAGS=-gencode=arch=compute_{0},code=sm_{0}".format(
-                        gpu_map[spec.variants["nvidia_arch"].value]
-                    ),
-                    "CFLAGS=-I{0}".format(spec["cuda"].prefix + "/include"),
-                    "CXXFLAGS=-I{0}".format(spec["cuda"].prefix + "/include"),
+                    f"--with-cuda={(spec["cuda"].prefix)}",
+                    f"NVCCFLAGS=-gencode=arch=compute_{gpu_map[spec.variants['nvidia_arch'].value]},code=sm_{gpu_map[spec.variants['nvidia_arch'].value]}",
+                    f"CFLAGS=-I{spec['cuda'].prefix + '/include'}",
+                    f"CXXFLAGS=-I{spec['cuda'].prefix + '/include'}",
                 ]
             )
         if "+rocm" in self.spec:
             args.extend(
                 [
                     "--enable-rocm",
-                    "--with-rocm={0}".format(spec["hip"].prefix),
+                    f"--with-rocm={0spec['hip'].prefix}",
```

Comment:
```suggestion
                    f"--with-rocm={spec['hip'].prefix}",
```

I think just a simple typo?